packages/forms/frontend/sirius-components-forms/src/propertysections/__tests__/__snapshots__/ListPropertySection.test.tsx.snap

```diff
@@ -469,7 +469,7 @@ exports[`should render a readOnly list from widget properties 1`] = `
             class="MuiTableCell-root MuiTableCell-body MuiTableCell-sizeSmall css-18jhh0s-cell css-dsuxgy-MuiTableCell-root"
           >
             <p
-              class="MuiTypography-root MuiTypography-body1  css-qh6f2j-style css-1gqyz35-MuiTypography-root"
+              class="MuiTypography-root MuiTypography-body1 css-qh6f2j-style css-1gqyz35-MuiTypography-root"
```

Comment:
Are those changes necessary? Similar question for the next file.

Reply:
I had an error on test execution with the previous version (with two spaces).
It should not have been modified previously, why two spaces?